tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py3, pep8
+envlist = py35, py36, pep8
```

Comment:
Please keep py3, I like "tox -e py3" to use system "python3" instead of a specific Python 3 version.